fairseq/data/denoising_dataset.py

```diff
@@ -26,7 +26,9 @@ def collate(
     def merge(key, left_pad, move_eos_to_beginning=False):
         return data_utils.collate_tokens(
             [s[key] for s in samples],
-            pad_idx, eos_idx, left_pad, move_eos_to_beginning,
+            pad_idx=vocab.pad(),
```

Comment:
I think this change from `pad_idx` to `vocab.pad()` may not be needed, since the function is called from DenoisingDataset.collater with `pad_idx=self.vocab.pad`, right?